src/__tests__/index.test.ts

```diff
@@ -45,4 +45,15 @@ describe('json-to-runtypes', () => {
       "
     `);
   });
+
+  it('Deals with invalid JS identifiers', () => {
+    expect(jsonToRuntypes({ '0': { foo: 'value' } })).toMatchInlineSnapshot(`
+      "import * as rt from \\"runtypes\\";
```

Comment:
Should upgrade to jest 29 for prettier snapshots 🙂 or at least pass the new config (https://jestjs.io/blog/2022/08/25/jest-29)